### PR TITLE
doc: add hybrid scenario document review edits

### PR DESCRIPTION
**Pull request by deb-intel**
Signed-off-by: Deb Taylor <deb.taylor@intel.com>

@lirui34 Under "Hybrid Scenario Startup Checking," please look at steps 2 d. through 2 h. I made edits but I am not sure if I added clarity or if I misunderstood what you mean. Thanks.

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: doc/tutorials/using_hybrid_mode_on_nuc.rst [hunk 1]
```diff
@@ -1,68 +1,60 @@
...
+
+   .. note:: The module ``/boot/zephyr.bin`` is the VM0(Zephyr) kernel file.
+      The param ``xxxxxx`` is VM0’s kernel file tag and must exactly match the
+      ``kernel_mod_tag`` of VM0 which is configured in the ``hypervisor/
+      scenarios/hybrid/vm_configurations.c`` file. The multiboot module ``/boot
+      /bzImage`` is the Service VM kernel file. The param ``yyyyyy`` is the
+      bzImage tag and must exactly match the ``kernel_mod_tag`` of VM1 in the
+      ``hypervisor/scenarios/hybrid/vm_configurations.c`` file.
+      The kernel command line arguments used to boot the Service VM is located
+      in the header file ``hypervisor/scenarios/hybrid/vm_configurations.h`` and is configured by `SOS_VM_BOOTARGS` MACRO.
```
Comment (dbkinder) [inline doc/tutorials/using_hybrid_mode_on_nuc.rst]: similarly, change ``is configured`` to ``are configured``, and I'd keep the line length < 80 here too.

File: doc/tutorials/using_hybrid_mode_on_nuc.rst [hunk 2]
```diff
@@ -72,31 +64,30 @@ Update Ubuntu GRUB to Boot hypervisor and Load Kernel Image
...
-   hypervisor, and the hypervisor will start VMs automatically.
+   hypervisor, and the hypervisor will start the VMs automatically.
 
 Hybrid Scenario Startup Checking
 ********************************
-#. Use these steps to verify the hypervisor is properly running:
+#. Use these steps to verify that the hypervisor is properly running:
 
-   a. Login ACRN hypervisor shell from serial console.
+   a. Log in to the ACRN hypervisor shell from the serial console.
```
Comment (dbkinder) [inline doc/tutorials/using_hybrid_mode_on_nuc.rst]: On the next line, change to ``Use the vm_list command to check the pre-launched ...``

File: doc/tutorials/using_hybrid_mode_on_nuc.rst [hunk 3]
```diff
@@ -72,31 +64,30 @@ Update Ubuntu GRUB to Boot hypervisor and Load Kernel Image
...
    #. Use the vm_list check pre-launched VM and Service VM are launched successfully.
 
 #. Use these steps to verify all VMs are running properly:
 
-   a. Use the ``vm_console 0`` to switch to VM0(Zephyr)’s console, it will show
-      a string of **Hello world! acrn**.
-   #. Use a :kbd:`Ctrl+Spacebar` to return to the ACRN hypervisor shell.
+   a. Use the ``vm_console 0`` to switch to VM0(Zephyr)’s console. It will display **Hello world! acrn**.
```
Comment (dbkinder) [inline doc/tutorials/using_hybrid_mode_on_nuc.rst]: Change to:
```
Use the ``vm_console 0`` command to switch to the VM0 (Zephyr) console.
```

File: doc/tutorials/using_hybrid_mode_on_nuc.rst [hunk 4]
```diff
@@ -72,31 +64,30 @@ Update Ubuntu GRUB to Boot hypervisor and Load Kernel Image
...
+   #. Enter :kbd:`Ctrl+Spacebar` to return to the ACRN hypervisor shell.
    #. Use the ``vm_console 1`` to switch to VM1(Service VM)’s console.
-   #. The VM1’s Service VM could boot up and login in.
-   #. SSH to VM1 and Launch post-launched VM2 by ACRN device model launch script.
-   #. Back to Service VM console, use a :kbd:`Ctrl+Spacebar` to return to the ACRN hypervisor shell.
-   #. Use the ``vm_console 2`` to switch to VM2(User VM)’s console.
-   #. The VM2 could boot up and login in.
+   #. Verify that the VM1’s Service VM can boot up and you can log in.
```
Comment (dbkinder) [inline doc/tutorials/using_hybrid_mode_on_nuc.rst]: Change the line above to:
```
Use the ``vm_console 1`` command to switch to the VM1 (Service VM) console.
```

File: doc/tutorials/using_hybrid_mode_on_nuc.rst [hunk 5]
```diff
@@ -72,31 +64,30 @@ Update Ubuntu GRUB to Boot hypervisor and Load Kernel Image
...
+   #. SSH to VM1 and launch the post-launched VM2 using the ACRN device model launch script.
```
Comment (dbkinder) [inline doc/tutorials/using_hybrid_mode_on_nuc.rst]: Change to lower-case ``ssh``

File: doc/tutorials/using_hybrid_mode_on_nuc.rst [hunk 6]
```diff
@@ -72,31 +64,30 @@ Update Ubuntu GRUB to Boot hypervisor and Load Kernel Image
...
+   #. Back to Service VM console, enter :kbd:`Ctrl+Spacebar` to return to the ACRN hypervisor shell.
+   #. Use the ``vm_console 2`` to switch to the VM2(User VM)’s console.
```
Comment (dbkinder) [inline doc/tutorials/using_hybrid_mode_on_nuc.rst]: Change to:
```
Use the ``vm_console 2`` command to switch to the VM2 (User VM) console.
```

**Comment by acrnsi** [comment]
OK to verify

**Pull request merged**